frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -262,6 +261,7 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                 cache.performanceEventsStartTime = null
                 cache.firstPaintDurationRow = null
             } else {
+                await breakpoint(1)
```

Comment:
Just making sure we only want a 1ms breakpoint here.

Reply:
It technically shouldn't matter since `actions.reportUsage(SessionRecordingUsageType.VIEWED)` should only be called once, but I'm just putting 1 here to avoid potential parallel requests. Differentiating this from the `actions.reportUsage(SessionRecordingUsageType.LOADED)` codepath should be important fix though🤞

Reply:
argh scratch the above, actually setting this breakpoint to 1 breaks a test :/